release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -42,6 +42,45 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### Language
 
+##### Module level complex variable support
+
+Tuple variable, Record variable and Error variable declaration is now allowed at module level. Unlike simple variables, complex variable declaration must have an initialization.
+Complex variable declaration cannot contain `isolated` or `configurable` qualifier.
+```ballerina
+// Module level tuple variable declaration
+[int, float] [a, b] = [1, 2.5];
+
+// Module level record variable declaration
+Person {name: Fname, married: Married} = {name: "Jhone", married: true};
+
+type Person record {|
```

Comment:
Shall we move this definition above the variable declaration that uses `Person`?

Reply:
I moved it down to co-occur the comment and the declaration.

Reply:
We can move this before L55, right?